code/modules/unit_tests/outfit_sanity.dm

```diff
@@ -39,14 +39,15 @@
 	back = /obj/item/storage/backpack
 	backpack_contents = list(/obj/item/clothing/mask/cigarette/cigar/havana)
 
-
-
 /datum/unit_test/outfit_sanity/Run()
 	var/datum/outfit/prototype_outfit = /datum/outfit
 	var/prototype_name = initial(prototype_outfit.name)
 	var/mob/living/carbon/human/H = allocate(/mob/living/carbon/human/consistent)
 
-	for (var/outfit_type in subtypesof(/datum/outfit))
+	var/list/outfitsToCheck = subtypesof(/datum/outfit)
+	outfitsToCheck ^= subtypesof(/datum/outfit/deathmatch_loadout)
+
+	for (var/outfit_type in outfitsToCheck)
```

Comment:
```suggestion
	var/list/outfits_to_check = subtypesof(/datum/outfit)
	outfits_to_check -= subtypesof(/datum/outfit/deathmatch_loadout)

	for (var/outfit_type in outfits_to_check)
```

1. we don't want _any_ subtypes of /datum/outfit/deathmatch_loadout, right? let's just cull them out entirely by using `-=` instead of `^=`
2. prefer to use `snake_case` naming of variables instead of `camelCase`

Reply:
yep we can just do that, and theres not really any reason to include the base dm loadout type so I'll make that adjustment.
i'm rusty with byond code, my job habits (camel case) are flooding in. thanks for the quality pass

Reply:
No worries